### ISSUE TITLE
Development halted?

### ISSUE THREAD

**Issue by twhitbeck** [dhg/Skeleton#275]
Hi @dhg 
I'm very appreciative of your work here with Skeleton. At the same time, I'm having a hard time embracing it going forward since I don't see any recent activity from you. I know life gets busy. Would you be open to adding some collaborators to the project so things can continue to move forward? There are a number of open pull requests that seem ripe for the picking.

Maybe a skeleton organization could even be created to further move this work into the hands of the community. Whatever you think is best. Thanks again for the great work

**Comment by dmur** [comment]
:+1: 

Would be happy to help out as a collaborator.

**Comment by benhinchley** [comment]
:+1: would also be happy to help out

**Comment by peterblazejewicz** [comment]
:+1: 

I think the @dhg  is on the go:
https://github.com/dhg/Redditate/pull/13#issuecomment-99363895

**Comment by apwinter** [comment]
Would like to see Skeleton keep improving, I love it's simplicity. Breaking out of 960 would be great

**Comment by albertmoreu** [comment]
@apwinter Just change the "max-width" value of ".container" in "skeleton.css".

.container {
  position: relative;
  width: 100%;
  max-width: <strike>960px</strike> <strong>1200px</strong>;
  margin: 0 auto;
  padding: 0 20px;
  box-sizing: border-box; }

I'm looking forward to hear from @dhg and see if he can push some interesting commits that contributors are suggesting.

**Comment by apwinter** [comment]
Thanks, I hadn't thought of that.....awesome

**Comment by apwinter** [comment]
Can someone help please? I'm developing a site for a client and in a section of a page I have 3 rows with 2 x one-half columns in each, Each row is wrapped in a 'container' and 'row'. On the tablet the 2 half columns are overlapping instead of going down onto individual rows. On the smart phone they are going in separate rows but not reducing to 85% of screen. It could have to do with a 'block' I'm using in C5 but as far as I can see those settings are set to max-widths/heights. Here is the page in development http://tjsfurniture.com.au/cms/   Any ideas?

**Comment by twhitbeck** [comment]
@apwinter github issues are not intended to be a support forum. You'd be better off submitting your question to stackoverflow.com

**Comment by mileswd** [comment]
:+1: 

I assume there's still no update on this situation?

**Comment by champsupertramp** [comment]
+1

**Comment by HolyThunder** [comment]
It would be really cool if this project didn't die. It's really awesome and lightweight, and I've used it numerous times. Pleeeaaase get some collaborators going!

**Comment by OneHunnid** [comment]
I would LOVE to see further development as well as it respond better for mobile devices.  I'd really like to see this grid stack columns and display them in 50% widths between 550px to 850px, instead of having 12 columns get smaller in width until it hits the break point where it stacks the content. I've tried hacking it to do this, but the rows are giving me issues specifically, adding margins which throw it off.

**Comment by LeoJavaAI** [comment]
Can those willing to help, fork the project and continue work there @dmur @benhinchley

**Comment by HolyThunder** [comment]
I was looking for a decent active fork of the project.+1 on starting one.

**Comment by mileswd** [comment]
@HolyThunder It seems that may *have* to be the outcome of this, as it is almost 3 months since this ticket was created yet there is still no response from @dhg.

Causing fragmentation through forking the project isn't ideal, but it seems to be the only option for continuing development.

+1

**Comment by dmur** [comment]
Unfortunately I'm shorter on cycles than when I originally posted this, so I don't think I'd be the ideal candidate to be a primary maintainer of a fork.

[Skeleton-Sass](https://github.com/whatsnewsaes/Skeleton-Sass) looks to be more active and accepting PRs.

**Comment by nervecentral** [comment]
Looks like the guy who did Skeleton-Sass had the same idea—check out https://github.com/whatsnewsaes/Boneless

**Comment by benhinchley** [comment]
@nervecentral Yeah I just took over that project, trying to tidy everything up at the moment

**Comment by lesterzone** [comment]
I'm using this project. :+1:

**Comment by benhinchley** [comment]
Is an active fork of this something others would be willing to work on? I am up for starting one up or if someone else wants to, I'm happy to help contribute to it, I am currently looking after [Boneless](https://github.com/whatsnewsaes/Boneless) and many of the ideas from that could be transferred to Skeleton. Thoughts??

**Comment by benhinchley** [comment]
I have forked this (well reforked it) [benhinchley/Skeleton](https://github.com/benhinchley/Skeleton) and have begun work on adding the changes made in PR's and issues on this. If some others are interested in working on it, we could create an org and manage the repo that way??

**Comment by twhitbeck** [comment]
I like the idea of there being an org. That way when this happens in the future the project would have a little more stability and the primary repo wouldn't need to move around.

It'd be nice to get some feedback from @dhg on all this.

**Comment by mileswd** [comment]
+1 for an org - this project needs a higher bus factor :bus:

**Comment by gld1982ltd** [comment]
Organization started. https://github.com/SkeletonFramework I invited you!

**Comment by benhinchley** [comment]
Thanks for creating this @gld1982ltd, I just forked Skeleton into the org

**Comment by gld1982ltd** [comment]
OK, @benhinchley , I created a dev branch to Skeleton for work to be done on. I created a pull request for your work. Let's continue this there.

**Comment by ghost** [comment]
I have started a few days a go a new Version of skeleton. https://github.com/oltmannsdaniel/skeleton-plus

**Comment by dan1elhughes** [comment]
@oltmannsdaniel I appreciate you trying to get word out of your project, but it's a bit unnecessary to revive dozens of old issues to post the link.

**Comment by ghost** [comment]
I only want to share the link. Nothing more

**Comment by cottsak** [comment]
I think this is the more appropriate link https://github.com/skeleton-framework/skeleton-framework
